### PR TITLE
Create an Utility Panel

### LINKED ISSUE #1505: Modus Utility Panel Component - Bare Bones
Modus Style guide:
https://modus.trimble.com/components/web/utility-panel/

Figma:
https://github.com/trimble-oss/website-modus.trimble.com/issues/565

### PR DESCRIPTION
**Pull request by ElishaSamPeterPrabhu**
## Description

- Developed an Utility component based on the guidelines from https://modus.trimble.com/components/web/utility-panel/

References 
Fixes #1505 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: stencil-workspace/src/components/modus-utility-panel/modus-utility-panel.scss
Comment (coliff) [inline stencil-workspace/src/components/modus-utility-panel/modus-utility-panel.scss:1]: need to run stylelint on this (will auto-sort the properties alphabetically)

File: stencil-workspace/src/components/modus-utility-panel/modus-utility-panel.tsx
Comment (coliff) [inline stencil-workspace/src/components/modus-utility-panel/modus-utility-panel.tsx:1]: needs `// eslint-disable-next-line @typescript-eslint/no-unused-vars` on the first line.

File: stencil-workspace/storybook/stories/components/modus-utility-panel/modus-utility-panel.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+          Navbar.
+        </p>
+      </div>
+    </div>
+  </div>
+</div>
+${setEventListeners('#defaultTemplate')}
+`;
+
+const setEventListeners = (containerId) => {
```
Comment (coliff) [inline stencil-workspace/storybook/stories/components/modus-utility-panel/modus-utility-panel.stories.tsx:94]: `'containerId' is defined but never used` - is that intentional?

Comment (ElishaSamPeterPrabhu) [reply stencil-workspace/storybook/stories/components/modus-utility-panel/modus-utility-panel.stories.tsx]: Its not and removed

File: stencil-workspace/src/components/modus-utility-panel/modus-utility-panel.tsx
Comment (ElishaSamPeterPrabhu) [reply stencil-workspace/src/components/modus-utility-panel/modus-utility-panel.tsx:1]: Added eslint

File: stencil-workspace/src/components/modus-utility-panel/modus-utility-panel.scss
Comment (ElishaSamPeterPrabhu) [reply stencil-workspace/src/components/modus-utility-panel/modus-utility-panel.scss:1]: Ran and changes the order

**Review by cjwinsor — CHANGES_REQUESTED** [review]

**Comment by cjwinsor** [comment]
Once my final 2 comments are resolved, this is good to merge. @ElishaSamPeterPrabhu @prashanthr6383 @coliff

File: stencil-workspace/storybook/stories/components/modus-utility-panel/modus-utility-panel-storybook-docs.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+</div>
+</div>
+<script>
+document.addEventListener('helpOpen', (e) => {
+const panel = document.querySelector('modus-utility-panel');
+panel.expanded = !panel.expanded;
+});
+
+
+</script>`
```
[the remark this answers is not in the document]
Comment (ElishaSamPeterPrabhu) [reply stencil-workspace/storybook/stories/components/modus-utility-panel/modus-utility-panel-storybook-docs.mdx:45]: Removed

**Review by coliff — CHANGES_REQUESTED** [review]
There is an accessibilty violation so we can't merge it as it is.
image

Can you fix it please?

**Comment by ElishaSamPeterPrabhu** [comment]
> There is an accessibilty violation so we can't merge it as it is. image
> 
> Can you fix it please?

@coliff 
This issue is present in navbar 
we can remove the navbar and add a button to trigger the uility panel or fix it in navbar

**Comment by coliff** [comment]
@ElishaSamPeterPrabhu if you can fix that a11y issue on the navbar separaetly then that'd be great!

**Review by coliff — APPROVED** [review]

**Pull request merged**